src/embind/embind.js

```diff
@@ -319,6 +319,12 @@ var LibraryEmbind = {
         }
     }
 
+#if SIDE_MODULE || MAIN_MODULE
```

Comment:
You don't need SIDE_MODULE here of below since side modules by definition, don't contains any JS library code.

Reply:
Ah, thanks!